mmedit/models/backbones/encoder_decoders/necks/aot_neck.py

```diff
@@ -0,0 +1,109 @@
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class AOTBlockNeck(nn.Module):
+    """Dilation backbone used in AOT-GAN model.
+    This implementation follows:
+    Aggregated Contextual Transformations for High-Resolution Image Inpainting
+    Args:
+        in_channels (int): Channel number of input feature.
+        dilation_rates (str): The dilation rates used
+        for AOT block. Default: 1+2+4+8.
+        num_aotblock (int): Number of AOT blocks. Default: 8.
+        act_cfg (dict): Config dict for activation layer, "relu" by default.
+        kwargs (keyword arguments).
+    """
+
+    def __init__(self,
+                 in_channels=256,
+                 dilation_rates='1+2+4+8',
+                 num_aotblock=8,
+                 act_cfg=dict(type='ReLU'),
+                 **kwargs):
+        super().__init__()
+
+        self.dilation_rates = list(map(int, list(dilation_rates.split('+'))))
+        self.num_aotblock = num_aotblock
+
+        self.model = nn.Sequential()
+
+        for i in range(self.num_aotblock):
+            self.model.add_module(
+                f'aotblock{i + 1}',
+                AOTBlock(
+                    in_channels=in_channels,
+                    dilation_rates=self.dilation_rates,
+                    act_cfg=act_cfg,
+                ))
```

Comment:
If the name `aotblock{i + 1}` is not necessary, maybe we can simplify it as 
```
self.model = nn.Sequential(
[AOTBlock(
      in_channels=in_channels,
      dilation_rates=self.dilation_rates,
      act_cfg=act_cfg,
  ) for _ in range(0, num_aotblock)]
)
```
Something like that.